e2e/app_test.go

```diff
@@ -3,8 +3,11 @@ package e2e
 import (
 	"fmt"
 	"strconv"
+	"strings"
 	"testing"
 
+	"gotest.tools/v3/assert"
+
```

Comment:
no newline. could you use `goimports` for formatting please?

Reply:
i am using `goimports`, but it does not remove empty lines and does only the sorting. thanks for pointing out.